### PR TITLE
Ports Bay's Journalist TV Drone

### PR DESCRIPTION
**Pull request by Neerti**
Ports https://github.com/Baystation12/Baystation12/pull/14422
Also ports entertainment frequency and prerequisite code.
Entertainment monitors now broadcast entertainment frequency, so you can watch and listen to the journalist.

### CODE REVIEW

File: code/controllers/communications.dm [hunk 1]
```diff
@@ -146,7 +148,7 @@ var/list/CENT_FREQS = list(ERT_FREQ, DTH_FREQ)
 var/list/ANTAG_FREQS = list(SYND_FREQ)
 
 //Department channels, arranged lexically
```
Comment (PsiOmegaDelta) [inline code/controllers/communications.dm]: This comment is now a lie.

**Comment by Anewbe** [comment]
Did you test this at all?

**Comment by Neerti** [comment]
Yes, everything appeared to work fine.

**Comment by Anewbe** [comment]
'Merica

**Pull request merged**